### PR TITLE
audacious: 3.5.2 -> 3.7.2

### PR DESCRIPTION
**Pull request by RamKromberg**
###### Motivation for this change

was missing a lot of the plugins and features.
###### Things done
- [x] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [x] NixOS
  - [ ] OS X
  - [x] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/audio/audacious/default.nix [hunk 1]
```diff
@@ -1,69 +1,79 @@
...
+
+  configureFlags = [ "--enable-statusicon" ];
 
-  # Here we build bouth audacious and audacious-plugins in one
+  # Here we build both audacious and audacious-plugins in one
   # derivations, since they really expect to be in the same prefix.
   # This is slighly tricky.
-  builder = builtins.toFile "builder.sh"
-    ''
-      # First build audacious.
```
Comment (binarin) [inline pkgs/applications/audio/audacious/default.nix]: Is this indentation change necessary? It make harder to see the actual changes.

Comment (globin) [inline pkgs/applications/audio/audacious/default.nix]: Adding `?w=1` to the url hides whitespace changes: https://github.com/NixOS/nixpkgs/pull/17543/files?w=1

Comment (RamKromberg) [inline pkgs/applications/audio/audacious/default.nix]: it wasn't necessary but with all the changes I figured it won't matter.

File: pkgs/applications/audio/audacious/default.nix [hunk 2]
```diff
@@ -1,69 +1,79 @@
...
-      (
-        source $stdenv/setup
-        genericBuild
-      )
+  builder = builtins.toFile "builder.sh" ''
```
Comment (fpletz) [inline pkgs/applications/audio/audacious/default.nix]: At some point this should be refactored. Maybe use two derivations for audacious and audacious-plugins? Fine to merge for now, though.

File: pkgs/applications/audio/audacious/default.nix [hunk 3]
```diff
@@ -1,69 +1,79 @@
...
+    sha256 = "0gxka0lp9a35k2xgq8bx69wyv83dvrqnpwcsqliy3h3yz6v1fv2v";
   };
 
-  buildInputs =
-    [ gettext pkgconfig glib gtk3 libmowgli libmcs libxml2 dbus_glib
-      libmad xorg.libXcomposite libogg libvorbis flac alsaLib libcdio
-      libcddb ffmpeg makeWrapper mpg123 neon faad2 gnome3.defaultIconTheme
-    ];
+  buildInputs = [
+    pkgconfig autoconf makeWrapper gettext glib gtk3 libmowgli
```
Comment (fpletz) [inline pkgs/applications/audio/audacious/default.nix]: Why did you add autoconf here? Still builds fine after removing it from buildInputs.

Comment (RamKromberg) [inline pkgs/applications/audio/audacious/default.nix]: @fpletz  You killed some plugins by taking off autotools... Sec I'll tell you which ones.
